website/pages/en/sunrise.mdx

```diff
@@ -1,233 +1,79 @@
-
-To assist teams that are transitioning subgraphs from the hosted service to the Graph Network, curation support is now available. If you require assistance with curation to enhance the quality of service, please send a request to the Edge & Node team at support@thegraph.zendesk.com and specify the subgraphs you need assistance with.
-
-Once your subgraph has reached adequate curation signal and other Indexers begin supporting it, the upgrade Indexer will gradually taper off, allowing other Indexers to collect indexing rewards and query fees.
+### What does this mean for Delegators?
 
-### Should I host my own indexing infrastructure?
+The upgrade Indexer offers a powerful opportunity for Delegators. As more subgraphs were upgraded from the hosted service to The Graph Network, Delegators benefited from the increased network activity.
 
-Running infrastructure for your own project is [significantly more resource intensive](/network/benefits/) when compared to using The Graph Network.
+### Did the Upgrade Indexer compete with existing Indexers for rewards?
 
-Additionally, The Graph Network is significantly more robust, reliable, and cost-efficient than anything provided by a single organization or team. Hundreds of independent Indexers around the world power The Graph Network, ensuring safety, security, and redundancy.
+No, the upgrade Indexer only allocates the minimum amount per subgraph and does not collect indexing rewards.
 
-That being said, if you’re still interested in running a [Graph Node](https://github.com/graphprotocol/graph-node), consider joining The Graph Network [as an Indexer](https://thegraph.com/blog/how-to-become-indexer/) to earn indexing rewards and query fees by serving data on your subgraph and others.
+It operates on an “as needed” basis, serving as a fallback until sufficient service quality is achieved by at least three other Indexers in the network for respective chains and subgraphs.
 
-### Should I use a centralized indexing provider?
+### How does this affect subgraph developers?
 
-If you are building in web3, the moment you use a centralized indexing provider, you are giving them control of your dapp and data. The Graph’s decentralized network offers [superior quality of service](https://thegraph.com/blog/qos-the-graph-network/), reliability with unbeatable uptime thanks to node redundancy, significantly [lower costs](/network/benefits/), and keeps you from being hostage at the data layer.
+Subgraph developers can query their subgraphs on The Graph Network almost immediately after upgrading from the hosted service or publishing from Subgraph Studio, as no lead time was required for indexing.
```

Comment:
I'm not sure this question/answer is useful as is. Maybe it would be better to explain the differences in deploying/publishing new subgraph versions between hosted service and Subgraph Studio? And also highlight what _didn't_ change: how to develop subgraphs itself (e.g. subgraph manifest, schema, mappings, etc.) should be exactly the same, as far as I know.